executor/slow_query.go

```diff
@@ -490,6 +491,34 @@ func getLineIndex(offset offset, index int) int {
 	return fileLine
 }
 
+// splitByColon split a line like "field:value field:value..."
+func splitByColon(line string) (fields []string, values []string) {
```

Comment:
@breeswish Added, but I don't think this func can handle cases that is not regular like "123:456:789"...

Reply:
@breeswish  @bb7133  @crazycs520  PTAL

Reply:
Then it means `Time: 2021-09-08T14:39:54.506967433+08:00` will fail?

Reply:
> Then it means `Time: 2021-09-08T14:39:54.506967433+08:00` will fail?

Nice catch, I hadn't thought of that before. 
But it should be split by ": " instead of ":", so it's still ok here.

Reply:
> > Then it means `Time: 2021-09-08T14:39:54.506967433+08:00` will fail?
> 
> Nice catch, I hadn't thought of that before.
> But it should be split by ": " instead of ":", so it's still ok here.

yep, I will add the case into UT.